webpack/shared.js

```diff
@@ -113,9 +113,14 @@ module.exports = {
 	},
 	resolve: {
 		extensions: [ '.ts', '.tsx', '.json', '.js', '.jsx' ],
-		modules: [ path.join( process.cwd(), 'client' ), 'node_modules' ],
+		modules: [
+			path.join( process.cwd(), 'client' ),
+			path.join( process.cwd(), 'multi-currency', 'client' ),
+			'node_modules',
+		],
 		alias: {
 			assets: path.resolve( process.cwd(), 'assets' ),
+			mccy: path.resolve( process.cwd(), 'multi-currency', 'client' ),
```

Comment:
As we discussed on the call, let's rename this to `multi-currency` in favor of readability over brevity.

Reply:
Good call. Fixed in [1f01897](https://github.com/Automattic/woocommerce-payments/pull/9314/commits/1f018971ec4ce84254f458b827033c9b528ffff5).